symbols/overlay29.yml

```diff
@@ -3927,6 +4040,17 @@ overlay29:
         Activates the Mud Sport or Water Sport condition on the dungeon floor for some number of turns.
         
         r0: water sport flag (false for Mud Sport, true for Water Sport)
+    - name: TryActivateWeather
```

Comment:
This is missing a function signature. Do we know enough to specify one in the headers?

Reply:
I am not entirely confident on what the booleans passed in are used for. So I didn't know what to call them parameters in the header. Should I still add it based upon my guesses?

Reply:
I usually add a signature as long as I've confirmed _how many_ parameters there are, and I know what the return type is. If I don't know the rest, I just copy it from Ghidra. So the minimal signature would be something like

```c
<return type> MyFunction(undefined param_1, ..., undefined param_n);
```

The rest can be filled in as and when they're figured out.